Bottom: 4e05cf41a02d82ea76178c24c8cdc35b7932a2fc
Top:    c62a01a0bf616e40a4954ddb9e62ae9c44de5ef6
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-12-29 17:50:52 +0800

Refresh of tuntap-xdp-transmission

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index dd12bfe..a5a1db6 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -406,7 +406,7 @@ static void vhost_net_disable_vq(struct vhost_net *n,
 }
 
 static int vhost_net_enable_vq(struct vhost_net *n,
-			        struct vhost_virtqueue *vq)
+				struct vhost_virtqueue *vq)
 {
 	struct vhost_net_virtqueue *nvq =
 		container_of(vq, struct vhost_net_virtqueue, vq);
